nifi-nar-bundles/nifi-network-bundle/nifi-network-nar/pom.xml

```diff
@@ -1,40 +1,37 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not sure about how this formatting changed, but it should be the way it was

Reply:
Ah eclipse did the code formatting i think.. Let me change it  back.